### DISCUSSION TITLE
Export/import faster between multiple tenants

### DISCUSSION THREAD

**Discussion by DashCarlos** [Micke-K/IntuneManagement#143] [Ideas]
In my case I use several different tenants, so I miss a function that allows me to export/import between tenants automatically, instead of switching account, export, switching account, import. 
Do you think it is possible to implement it?.

Thank you for your wonderful contribution.

**Comment by Micke-K** [comment]
Hello!

I've been thinking about this for a long time. I would have to re-write a few things to get this to work. 

I gave a couple of requests of using app to login and that might start the process of implementing this. 

Cheers!

**Reply by DashCarlos** [reply]
Great news! Thank you
